MvsSln/Core/Guids.cs

```diff
@@ -65,8 +68,7 @@ public static class Guids
         /// <returns></returns>
         public static ProjectType ProjectTypeBy(string guid)
         {
-            return ProjectTypeGuids.Where(p => p.Value == guid)
-                    .Select(p => p.Key).First();
+            return ProjectTypeGuids.FirstOrDefault(p => p.Value == guid).Key;
```

Comment:
@3F implementing the lookup like this should default to `ProjectType.Unknown` since `FirstOrDefault()` returns the default value of `KeyValuePair<ProjectType, string>` which is a struct that should then initialize with the default `ProjectType` value. (see: https://stackoverflow.com/a/21404305/1662317)

Reply:
Yes, this is it, thank you. 
Seems it was a copy-paste for initial draft 2.2